page-objects/clothesPage.ts

```diff
@@ -1,11 +1,42 @@
 import { Page } from "@playwright/test";
 
-export class clothesPage {
 
-readonly page: Page
 
```

Comment:
Comment is related to all function in this page:
- function name should be descriptive in the way that reader can figure out what that function do (e.g. if you have only to locate element the 'getElementInput' is ok, but if you have to click() or fill() inside function than function name should explain that)